meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/uz/boot.cmd

```diff
@@ -20,15 +20,30 @@ setenv bootloader_s_image ${bootloader_image}
 setenv bootloader2_image "u-boot.itb"
 setenv bootloader2_s_image ${bootloader2_image}
 
-setenv update_primary_image 'echo "${fio_msg} writing ${image_path} ..."; mmc dev ${devnum} && fatwrite mmc ${devnum}:${bootpart} ${loadaddr} boot0001.bin ${filesize}';
-setenv update_secondary_image 'echo "${fio_msg} writing ${image_path} ..."; mmc dev ${devnum} && fatwrite mmc ${devnum}:${bootpart} ${loadaddr} boot0002.bin ${filesize}';
+if test "${modeboot}" = "qspiboot"; then
+	setenv qspi_bootloader_offset 0x0
+	setenv qspi_bootloader_s_offset 0x50000
+	setenv qspi_bootloader2_offset 0x100000
+	setenv qspi_bootloader2_s_offset 0xaa0000
 
-setenv update_primary_image2 'echo "${fio_msg} writing ${image_path} ..."; mmc dev ${devnum} && fatwrite mmc ${devnum}:${bootpart} ${loadaddr} u-boot0001.itb ${filesize}';
-setenv update_secondary_image2 'echo "${fio_msg} writing ${image_path} ..."; mmc dev ${devnum} && fatwrite mmc ${devnum}:${bootpart} ${loadaddr} u-boot0002.itb ${filesize}';
+	setenv update_primary_image 'echo "${fio_msg} writing ${image_path} ..."; sf probe; sf update ${loadaddr} ${qspi_bootloader_offset} ${filesize}';
```

Comment:
Wouldn't it be better to generalize setting `update_*` commands to avoid code duplication?
Like
```
if test "${modeboot}" = "qspiboot"; then
...
   setenv update_cmd 'sf probe; sf update ${loadaddr}'
   setenv image_pri1 '${qspi_bootloader_offset}'
   setenv image_sec1 '${qspi_bootloader_s_offset}'
...
else
  setenv update_cmd 'mmc dev \${devnum} && fatwrite mmc \${devnum}:\${bootpart} \${loadaddr}'
  setenv image_pri1 'boot0001.bin'
  setenv image_sec1 'boot0002.bin'
fi
setenv update_primary_image 'echo "${fio_msg} writing ${image_path} ..."; ${update_cmd} ${image_pri1} ${filesize}';
setenv update_primary_image 'echo "${fio_msg} writing ${image_path} ..."; ${update_cmd} ${image_sec1} ${filesize}';
...
```

Reply:
sure, nice idea, will do